src/lib/fetch.ts

```diff
@@ -1,19 +1,37 @@
 /* eslint-disable no-param-reassign */
-const { debug } = require('./logger');
-const fetch = require('node-fetch');
-const isGHA = require('./isGitHub');
-const mime = require('mime-types');
-const pkg = require('../../package.json');
-const APIError = require('./apiError');
+import type { Headers } from 'form-data';
```

Comment:
I'm not sure why this got pulled from `form-data` when I imported but because we aren't doing `Headers.set()` where we're setting headers I'm swapping this out with `Record<string, string>`.

Reply:
jk I'm moving our headers usage over to using the `Headers` API.